web-external/src/style/full-viewport.styl

```diff
@@ -1,24 +1,23 @@
-
 html
-  height: 100%
+  height 100%
 
 body
-  position: absolute
-  top: 0
-  bottom: 0
-  left: 0
-  right: 0
+  position absolute
+  top 0
+  bottom 0
+  left 0
+  right 0
 
 #g-global-nav-container
-  z-index: 1
+  z-index 1
 
-#g-app-footer-container
-  position: fixed
-  bottom: 0
-  left: 0
-  right: 0
+DISABLE#g-app-footer-container
+  position fixed
+  bottom 0
+  left 0
+  right 0
 
-#g-app-body-container
-  height: 95%
-  padding-bottom: 88px
+DISABLE#g-app-body-container
+  height 95%
+  padding-bottom 88px
```

Comment:
(and to do so in such a way as to make all body content viewable at the right scroll position).  I thought the `padding-bottom` attribute should have taken care of the footer area?  Am I missing something?  Is the padding just not enough?